packages/saber/lib/index.js

```diff
@@ -30,6 +30,8 @@ class Saber {
       // Before all user plugins have been applied
       beforePlugins: new AsyncSeriesHook(),
       filterPlugins: new SyncWaterfallHook(['plugins']),
+      // Extend saber config
+      getSaberConfig: new AsyncSeriesHook(['config']),
```

Comment:
should rename to `modifySaberConfig` then if nothing needs to be returned in the hook

Reply:
Yes, logical